specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/iotAlerts.json

```diff
@@ -0,0 +1,284 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2020-08-06-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{scope}/providers/Microsoft.Security/iotAlerts": {
```

Comment:
These are moving from being children of iotSecuritySolutions to their own resource type?  That is a pretty big change in behavior from the previous API version and should probably go through a breaking change review.  What is the driving force behind the change?

Reply:
Yes, these are moving from being children of iotSecuritySolutions to their own resource type. We are moving away from the "solution" concept (which is being deprecated internally) as we concluded that for us, solution == iot hub. So instead we will have new APIs as extension resource at  iot hub level and/or subscription level.

In the future we plan to decommision the solution level APIs completly.

Reply:
That is a large change to the structure of your resources.  Please describe what the migration path is for the existing resources, what happens when users use the old api-version that is still under iotSecuritySolutions, etc...  I.e. how the old and the new interplay between each other.  Since there wasn't much progress on this review last week @majastrz will take it forward from here

Reply:
For the meanwhile, and until otherwise decided, both APIs co-exists - current API will still be supported, and the API in this PR is in addition to existing API. There is no migration path yet as we are not sure how/when/if we'll decommission the current, solution based API

Reply:
Overall the API looks fine from the ARM side. However, I agree with @pilor that this should be looked at by the breaking changes committee. I don't think you need to follow the full process. I'd start with a mail thread to azbreakchangereview.

Reply:
Email sent. Since the API is backwards compatible (new endpoints are only in 2020-08-06-preview version, and previous versions of the API continue to be supported as they were) I'm not sure whats the crictiria for breaking change review - what is considered breaking change here and in general?

Reply:
The general guidelines are located at https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md.